server/webapp/WEB-INF/rails.new/app/assets/javascripts/jquery.environmentVariableEdit.js

```diff
@@ -0,0 +1,65 @@
+(function ($) {
+  $.resetEnvironmentVariableOn = function (options) {
```

Comment:
I do not see any tests for this javascript. Could you write them and update the PR.